docs/source/technical_tutorials/access/copc_access .ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>Pls add explanation for adding this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>